scraper/src/v2_json_downloader.py

```diff
@@ -11,26 +14,49 @@ def generate_v2_file(url, data, name=""):
 
     f = open("data/" + life_file_name_v2, 'w')
     print("Adding file: "+life_file_name_v2)
+    print(len(data["data"]))
     f.write(json.dumps(data))
     f.close()
 
 
 def parseInfo(info_type, data):
-    active_district_data_v2_data = {
-        "data": []
-    }
+    s = set()
     json_array = json.loads(data)
-    for info_type_json in json_array:
-        active_district_json = dict(ambulance="false", contact="false", district="", doctor="false", helpline="false",
-                                    hospitals="false", medicine="false", oxygen="false", state="")
-        active_district_json["state"] = info_type_json["state_name"]
-        active_district_json["district"] = info_type_json["district_name"]
```

Comment:
@vigneshhari do we need this now?
I think you have changed the output format to `state` and `district`